### PR TITLE
Updated download releases

### PR DESCRIPTION
**Pull request by gcolliso**
Updated download releases
Changed favicon to 4-color icon
Updated typo in About page tab description
Added Ginger Collison to Upcoming events Kubecon Who

### CODE REVIEW

**Comment by aricart** [comment]
One question, why add a link directly to the website when we should just be linking the release page? This will require less maintenance, point to the changes, etc.

**Comment by kozlovic** [comment]
@aricart We had this discussion in the past I believe. I am personally not keen in having individual links in there, but I group decided that it would be nice to be able to download directly from nats.io.

**Comment by aricart** [comment]
Maybe there's a way we can make it better. In reality, the downloads/etc are all driven by the version number. So we can generate that and only require the version number change and insure less work and consistency.

File: data/releases.yaml [hunk 1]
```diff
@@ -653,31 +653,31 @@ Releases:
...
+    url: https://github.com/nats-io/nats-streaming-server/releases/download/v0.9.2/nats-streaming-server-v0.9.2-linux-arm6.zip
+  - name: nats-streaming-server-v0.9.2-linux-arm7.zip
+    url: https://github.com/nats-io/nats-streaming-server/releases/download/v0.9.2/nats-streaming-server-v0.9.2-linux-arm7.zip
+  - name: nats-streaming-server-v0.9.2-linux-arm64.zip
+    url: https://github.com/nats-io/nats-streaming-server/releases/download/v0.9.2/nats-streaming-server-v0.9.2-linux-arm64.zip
+  - name: nats-streaming-server-v0.9.2-windows-386.zip
+    url: https://github.com/nats-io/nats-streaming-server/releases/download/v0.9.2/nats-streaming-server-v0.9.2-windows-386.zip
+  - name: nats-streaming-server-v0.9.2-windows-amd64.zip
+    url: https://github.com/nats-io/nats-streaming-server/releases/download/v0.9.2/nats-streaming-server-v0.9.2-windows-amd64.zip
   - name: nats-streaming-server-v0.8.0-beta-darwin-amd64.zip
```
Comment (kozlovic) [inline data/releases.yaml]: Remove from here down to line before Docker Image link please.

Comment (gcolliso) [reply data/releases.yaml]: Ok - so remove all beta 0.8.0 releases. Do you want the beta release information removed from the text box at the top left of the page too?

**Pull request merged**